src/sequelize.js

```diff
@@ -1276,11 +1284,11 @@ Sequelize.Deferrable = Deferrable;
 Sequelize.prototype.Association = Sequelize.Association = Association;
 
 /**
- * Provide alternative version of `inflection` module to be used by `Utils.pluralize` etc.
+ * Provide alternative version of `inflection` module to be used by `pluralize` etc.
  *
  * @param {object} _inflection - `inflection` module
  */
-Sequelize.useInflection = Utils.useInflection;
+Sequelize.useInflection = useInflection;
```

Comment:
Do we still want this?

Reply:
I suppose it's a way to let users customize the inflection module. It doesn't seem to be documented and I'm not a huge fan of doing this but I'd say let's keep it for now

Reply:
Can't they do that on the inflection module itself? Especially if we're removing the exports for singularize/pluralize/underscore

Reply:
I propose to look into useInflection in another PR. I don't want to remove something that may have a valid use case before we provide a cleaner alternative. Let's make a PR dedicated to removing useInflection